src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.utils.DukeException;
+import duke.utils.DukeFileHandler;
+import duke.utils.TaskList;
+import duke.utils.Ui;
+
+
+public class DeleteCommand extends Command {
+    private final int entryDelete;
```

Comment:
Perhaps we can explore adopting a more expressive variable name here such as IndexToDelete to explain the purpose of this variable at a higher level of detail?